fivenines_update.sh

```diff
@@ -1,4 +1,45 @@
 #!/bin/bash
+# This script is used to update the fivenines agent, and run by root
+
+# stop the agent
+sudo systemctl stop fivenines-agent.service
+
+# if the home directory of user "fivenines" is /home/fivenines (which is the old location), migrate user's home directory to /opt/fivenines
+if [ "$(getent passwd fivenines | cut -d: -f6)" == "/home/fivenines" ]; then
+        echo "Migrating fivenines.io's working directory from /home/fivenines to /opt/fivenines"
+        # if /opt/fivenines exists, or /home/fivenines not exists, exit
+        if [ -d /opt/fivenines ] || [ ! -d /home/fivenines ]; then
+                echo "Error: /opt/fivenines already exists or /home/fivenines does not exists"
+                exit 1
+        fi
+        sudo usermod -m -d /opt/fivenines fivenines
+
+        # Reinstall the agent due to path changes
+        echo "Reinstalling the fivenines_agent package"
```

Comment:
Do we really need that step as the home folder has been moved ?

Reply:
Yup. I tried to keep everything, but the pipx and modules saved the previous location as path references.